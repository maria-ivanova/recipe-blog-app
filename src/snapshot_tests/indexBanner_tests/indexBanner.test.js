import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import IndexBanner from '../../components/indexBanner.js';

describe('Index Banner Component', () => {
    it('should render layout', () => {
        const tree = renderer.create(
            <Routing>
                <IndexBanner/>
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})