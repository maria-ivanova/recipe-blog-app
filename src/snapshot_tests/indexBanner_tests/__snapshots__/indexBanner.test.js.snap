// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Banner Component should render layout 1`] = `
<section
  className="sec banner_sec"
>
  <div
    className="banner_item"
  >
    <h2
      className="title"
    >
      Сготви бързо и лесно с 
      <br />
      <span>
        Spatulas.bg
      </span>
    </h2>
    <a
      className="btn btn"
      href="/recipes/Всички рецепти"
      onClick={[Function]}
    >
      Всички рецепти
    </a>
  </div>
</section>
`;
